catalog/core/catalog-core-camelcomponent/pom.xml

```diff
@@ -100,6 +100,10 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.hazelcast</groupId>
+            <artifactId>hazelcast</artifactId>
```

Comment:
If we want to use this in a cluster, it look slike EHCache would be our only option. If we don't swap it out in this PR, we should just make a ticket to do so. No sense in sitting on something we can't use in that environment.